setup.py

```diff
@@ -8,10 +8,12 @@ def read(fname):
 
 setup(
     name = "rsplan",
-    version = "1.0.8",
+    version = "1.0.9",
     author = "Built Robotics",
     author_email = "tarakapoor9@gmail.com",
```

Comment:
should this be engineering@builtrobotics.com

Reply:
Yes